### PR TITLE
Allow target field entity to exist in parent class as private

### PR DESCRIPTION
**Pull request by szymach**

### CODE REVIEW

File: tests/FSi/DoctrineExtensions/Tests/Uploadable/Fixture/Inheritance/Employee.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+/**
+ * @ORM\Entity
+ */
+class Employee extends Person
+{
+    /**
+     * @ORM\Column(nullable=true)
+     * @FSi\Uploadable(targetField="file")
+     */
+    private $filePath;
```
Comment (chives) [inline tests/FSi/DoctrineExtensions/Tests/Uploadable/Fixture/Inheritance/Employee.php]: I think some problem shows up when this field (and/or `$file`) is defined in parent class

File: lib/FSi/DoctrineExtensions/Uploadable/UploadableListener.php [hunk 2]
```diff
@@ -622,4 +623,23 @@ private function generateNewKey(KeymakerInterface $keymaker, $object, $property,
 
         return $newKey;
     }
+
+    /**
+     * @param string $class
+     * @param string $property
+     * @return boolean
+     */
+    private function assertPropertyExistsInClassTree($class, $property)
```
Comment (chives) [inline lib/FSi/DoctrineExtensions/Uploadable/UploadableListener.php]: `doesPropertyExistInClassTree`

Comment (szymach) [inline lib/FSi/DoctrineExtensions/Uploadable/UploadableListener.php]: `propertyExistsInClassTree`?

Comment (chives) [inline lib/FSi/DoctrineExtensions/Uploadable/UploadableListener.php]: whatever... `assert` suggest that this method throws an exception

**Comment by szymach** [comment]
@rn0 @JarekW

**Pull request merged**